aws/resource_aws_instance.go

```diff
@@ -254,6 +254,12 @@ func resourceAwsInstance() *schema.Resource {
 				Optional: true,
 			},
 
+			"hibernation": {
```

Comment:
Typically we will want to follow the API structure in Terraform (e.g. it would look like the below), however this resource already goes against that in other attributes and this implementation won't prevent that later potential refactoring, so merging this as-is.

```hcl
resource "aws_instance" "example" {
  # ... other configuration ...
  hibernation_options {
    configured = true
  }
}
```